src/Shared/Model/Metadata/NuGetPackageVersionMetadata.cs

```diff
@@ -90,8 +90,11 @@ public record NuGetPackageVersionMetadata : IManagerPackageVersionMetadata
     [JsonProperty(PropertyName = JsonProperties.Listed)]
     public bool IsListed { get; init; }
 
+    /// <summary>
+    /// Gets if the package is verified within the prefix it is a part of.
+    /// </summary>
     [JsonProperty(PropertyName = JsonProperties.PrefixReserved)]
-    public bool PrefixReserved { get; init; }
+    public bool Verified { get; init; }
```

Comment:
I don't think this is a good name for this field. Verified on its own to me would indicate that we have done some verification of the package. Prefix reserved seems to be much more accurately describing the state.